lib/layout.js

```diff
@@ -135,7 +135,7 @@ const getPreferenceType = (preference) => (
 /* Dimension Fit Checks */
 
 const fitWithinChecker = (dimension) => (domainSize, itemSize) => (
-  domainSize[dimension] > itemSize[dimension]
+  domainSize[dimension] >= itemSize[dimension]
```

Comment:
Well, when the dimension of the pop up are 100%, the popover does not fit, while instead it should  fit.
Is it a problem that I fix other stuff in the same PR?